docoptTests/DocoptTestCasesTests.swift

```diff
@@ -65,7 +60,12 @@ class DocoptTestCasesTests: XCTestCase {
     
     private func fixturesFileContents() -> String {
         if let filePath = self.fixturesFilePath() {
-            let fileContents = String(contentsOfFile: filePath, encoding: NSUTF8StringEncoding, error: nil)
+            let fileContents: String?
+            do {
+                fileContents = try String(contentsOfFile: filePath, encoding: NSUTF8StringEncoding)
+            } catch _ {
+                fileContents = nil
+            }
             return fileContents!
```

Comment:
I guess this'll crash if fileContents is `nil`. So it should be ok to use `try!` without `do-catch`...